tests/sources/tools/perception/object_detection_2d/detr/test_detr.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2020-2021 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import unittest
+import shutil
+import os
+import torch
+import warnings
+from torch.jit import TracerWarning
+from opendr.engine.datasets import ExternalDataset
+from opendr.perception.object_detection_2d.detr.detr_learner import DetrLearner
+from PIL import Image
+
+
+DEVICE = "cuda:0" if torch.cuda.is_available() else "cpu"
+
+print("Using device:", DEVICE)
+print("Using device:", DEVICE, file=sys.stderr)
+
+
+def rmfile(path):
+    try:
+        os.remove(path)
+    except OSError as e:
+        print("Error: %s - %s." % (e.filename, e.strerror))
+
+
+def rmdir(_dir):
+    try:
+        shutil.rmtree(_dir)
+    except OSError as e:
+        print("Error: %s - %s." % (e.filename, e.strerror))
+
+
+class TestDetrLearner(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Please add a log with the test name, something like
```suggestion
    def setUpClass(cls):
        print("\n\n**********************************\nTEST Object Detection DETR Learner\n"
              "**********************************")
```